### PR TITLE
CPLAT-6852: Add disposable type name(s)

### PR DESCRIPTION
**Pull request by olesiathoms-wk**
Some of the classes that extend or mixin `Disposable` missing `disposableTypeName` override.
     This PR will add missing overrides, where it's necessary

### CODE REVIEW

**Comment by aviary-wf** [comment]
## Security Insights
No security relevant content was detected by automated scans.

## Action Items
* Review PR for [security impact](https://wiki.atl.workiva.net/display/SECURITY/Development+Security+Review+Guidelines); comment "security review required" if needed or unsure
* Verify [`aviary.yaml`](https://dev.workiva.net/docs/teams/information-security/aviary#configuring-aviary) coverage of security relevant code
---
_Questions or Comments? Reach out on Slack: #support-infosec._

File: lib/dart/.dart_tool/pub/deps/debug/w_transport/lib/src/web_socket/vm/web_socket.dart [hunk 1]
```diff
@@ -0,0 +1,98 @@
+// Copyright 2015 Workiva Inc.
```
Comment (smaifullerton-wk) [inline lib/dart/.dart_tool/pub/deps/debug/w_transport/lib/src/web_socket/vm/web_socket.dart]: I don't think we want to commit this file?

Comment (olesiathoms-wk) [reply lib/dart/.dart_tool/pub/deps/debug/w_transport/lib/src/web_socket/vm/web_socket.dart]: oh yeah that was an accident, I'll fix it

**Comment by brianshannan-wf** [comment]
+1

**Comment by kevinfrommelt-wf** [comment]
+1

**Comment by brianshannan-wf** [comment]
@Workiva/release-management-p

**Pull request merged**